app/src/main/java/org/simple/clinic/deeplink/DeepLinkActivity.kt

```diff
@@ -131,7 +131,7 @@ class DeepLinkActivity : AppCompatActivity(), DeepLinkUiActions {
     finish()
   }
 
-  override fun navigateToPatientSummaryWithTeleconsultLog(patientUuid: UUID, teleconsultRecordId: UUID?) {
+  override fun navigateToPatientSummaryWithTeleconsultLog(patientUuid: UUID, teleconsultRecordId: UUID) {
```

Comment:
Will this change still work if a medical officer receives a teleconsult request from the current implementation (without the teleconsult record ID)?

We would still want the older flow to work if the medical officer has updated to the new version but the nurse has not.

Reply:
Yes, there is a different function for that called `navigateToPatientSummary`. So the older flow will continue to work as expected.